plugins/UsersManager/tests/System/expected/test___UsersManager.getUsersWithSiteAccess_3_admin_when_superuseraccess.xml

```diff
@@ -19,6 +20,7 @@
 		
 		<invited_by />
 		
+		<invite_link_token />
```

Comment:
The invite token are for sure nothing that should be returned in any API response. 
The are even stored hashed in database, so there is no use for having that at all. Those should be hiddne like the password and other fields. e.g. here https://github.com/matomo-org/matomo/blob/45058209303f51216367f07fde54f83a6928e1e6/plugins/UsersManager/Repository/UserRepository.php#L143-L153

Reply:
removed token field